styles/components/map.scss

```diff
@@ -0,0 +1,7 @@
+
+.c-map {
```

Comment:
Component global layout (position, size) is not internal to the component, and should not be defined at component level, but at page layout level.